eml/screens/courses/CourseOverviewScreen.jsx

```diff
@@ -134,9 +135,7 @@ export default function CourseOverviewScreen({ route }) {
 									{course.title}
 								</Text>
 								{/* TODO: Button to download course should be implemented */}
-								<TouchableOpacity>
-									<MaterialCommunityIcons name="cloud-download-outline" size={30} color="gray"/>
-								</TouchableOpacity>
+								<DownloadCourseButton course={course} disabled={true}/>
```

Comment:
if you set disabled to false the download button will work, but i suppose its fine to leave it disabled for now since you can already download it on the coursecard and perhaps download in here was meant to download specific sections/lectures